app/connectors/API1811/FinancialDataConnector.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors.API1811
```

Comment:
I'm glad we made a separate package for API1811 files, but can you also do the same for the existing API1166 files that have the same name? Looks like that should only be the connector and HTTP parser. I'm finding it confusing we have for example `FinancialDataConnector` and `FinancialDataConnector.API1811` and I think if they both had the API number in the package it would be much clearer (which is also what the `models` package is doing now, for reference).